apidef/oas/server.go

```diff
@@ -58,3 +60,20 @@ func (lp *ListenPath) ExtractTo(api *apidef.APIDefinition) {
 	api.Proxy.ListenPath = lp.Value
 	api.Proxy.StripListenPath = lp.Strip
 }
+
+type ClientCertificates struct {
+	// AllowList is the list of client certificates which are allowed.
+	AllowList []string `bson:"allow_list" json:"allow_list"`
+	// Enabled enables static mTLS for the API.
+	Enabled bool `bson:"enabled,omitempty" json:"enabled,omitempty"`
```

Comment:
`Enabled` should sit at the beginning of the struct and the `Fill` and `ExtractTo` funcs should keep the same order for consistency.